proto/compiler.bzl

```diff
@@ -36,22 +36,28 @@ def go_proto_compile(ctx, compiler, lib, importpath):
   if plugin_base_name.startswith(_protoc_prefix):
     plugin_base_name = plugin_base_name[len(_protoc_prefix):]
   args = ctx.actions.args()
+  args.add(["-protoc", compiler.protoc.path])
+  options = compiler.options + ["import_path="+"goats"]
```

Comment:
Debug variable?

Reply:
Accidentally committed an unfinished experiment to the wrong branch... thanks for spotting